src/main/java/com/coveros/selenified/element/check/Matches.java

```diff
@@ -150,18 +150,18 @@ default String checkText(int row, int col, String pattern, double waitFor, doubl
      * @param timeTook        - the amount of time it took for wait for something (assuming we had to wait)
      * @return String: the actual value of the element. null will be returned if the element isn't present or an input
      */
-    default String checkValue(String expectedPattern, double waitFor, double timeTook) {
+    String checkValue(String expectedPattern, double waitFor, double timeTook) {
         // record the action and check this is an input element
-        if (!isPresentInput(getElement().prettyOutputStart() + " is not an input on the page", waitFor)) {
+        if (!isPresentInput(this.element.prettyOutputStart() + " is not an input on the page", waitFor)) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " is not an input on the page" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)